package.json

```diff
@@ -46,6 +46,21 @@
           "type": "boolean",
           "default": true,
           "description": "Indicate wether the variables values in the tooltip should be separated with dashes"
+        },
+        "templateFinder.display.colorAllMatching": {
+          "type": "string",
+          "default": "rgba(0,255,0,0.75)",
+          "description": "Variable highlight color if all match"
+        },
+        "templateFinder.display.colorSomeMatching": {
+          "type": "string",
+          "default": "rgba(255,140,0,0.75)",
+          "description": "Variable highlight color if some match"
+        },
+        "templateFinder.display.colorNoneMatching": {
+          "type": "string",
+          "default": "rgba(255,0,0,0.9)",
```

Comment:
Could you add "format": "color", so that vscode pops up the color picker in the settings file ?
It only recognizes hex colors, so as default values I'd suggest:
green: "#00cf00"
orange: "#e77f00"
red: "#ef0000"

Reply:
I added `"format": "color"` to package.json and removed ` "type": "string"` so "Edit in settings.json" would appear.

The color picker now shows up when editing the settings.json.
![image](https://user-images.githubusercontent.com/5135902/152645264-4dbcf7fb-80f9-4b61-9bba-19e1305445ee.png)

I was unable to get the color picker right within the UI settings page. Is this the correct way?

Reply:
I don't think that's possible, I've never seen a color picker in the UI settings editor. You could have left the "type": "string" though, That way the value still shows in the UI editor, even if the color picker doesn't.

Reply:
Sounds good, I have pushed the updated package.json.